### PR TITLE
Discord and Alexa notification services. Third attempt, fixed spacing deletion issue. 

### PR DESCRIPTION
**Pull request by jusplainmike**
Added two services: discord, alexa-notify-me

Alexa notifications are providing using Notify-Me voice app. 
http://www.thomptronics.com/notify-me

Discord notifications using built in webhooks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -574,6 +594,25 @@ targets = {
 Requires:
 * Python [dbus](http://www.freedesktop.org/wiki/Software/DBusBindings/#Python) bindings
 
+### `discord`
+
+The `discord` service provides notifications through discords webhooks.
```
Comment (jpmens) [inline README.md]: There's an apostrophe missing here: `discord's`

File: README.md [hunk 2]
```diff
@@ -574,6 +594,25 @@ targets = {
...
+
+![Discord Webhook](assets/discord.png)
```
Comment (jpmens) [inline README.md]: Are you aware that your screenshot displays `mmqt` several times? This is not a problem for me, just pointing out. (I assume you mean `mqtt` :-) )

File: services/discord.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    ''' expects (URL) in addrs '''
+
+    srv.logging.debug("*** MODULE=%s: service=%s, target=%s", __file__, item.service, item.target)
+
+    try:
+        srv.logging.debug("Sending to discord webhook")
+
+	url = item.addrs[0]
+
+	payload = "------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"content\"\r\n\r\n" + item.message + "\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"username\"\r\n\r\n" + item.topic + "\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW\r\nContent-Disposition: form-data; name=\"tts\"\r\n\r\nfalse\r\n------WebKitFormBoundary7MA4YWxkTrZu0gW--"
```
Comment (jpmens) [inline services/discord.py]: Does discord not support JSON payloads with `Content-type: application/json`? This looks rather brittle to me, particularly if we just interpolate `item.message` and `item.topic` like that ...

Comment (jusplainmike) [reply services/discord.py]: I will rewrite this as a general webhook service. I couldn't remember what issues I was having when I made this earlier this year.

File: README.md (hunk 2 above)
Comment (jusplainmike) [reply README.md]: I had two channels for mqtt, didn't notice the type thanks.

File: services/discord.py (hunk 3 above)
Comment (jusplainmike) [reply services/discord.py]: https://github.com/kyb3r/dhooks

I found this which I may use. If so I'll make a general webhook service as well.

**Comment by jusplainmike** [comment]
Kinda new to this whole git thing really. I made the changes to my repo, do I need to close this request and make a new one?

**Comment by jpmens** [comment]
No, that's fine here. You've made changes and have comitted and pushed them so I see them. Tell me when you're done and I'll press the necessary button to merge. (After that point you won't be able to push to this pull-request but will have to open a new one.)

**Comment by jusplainmike** [comment]
No problem. Glad I am able to help develop such a useful tool.

**Pull request merged**